test/SourceKit/CursorInfo/issue-62457-symbol-graph-crash.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %sourcekitd-test -req=cursor -req-opts=retrieve_symbol_graph=1 -pos=8:13 %s -- %s
+
+struct AnyError: Swift.Error {
+  let error: Swift.Error
+}
+
+func test(lhs: AnyError) {
+  lhs.error._code
```

Comment:
Could you add some checks that the symbol graph looks reasonable? Could name the test something like `private-stdlib-symbol-graph.swift`.

Reply:
Ah I see Alex beat me to that. I really do think the response is more relevant than the fact it crashed here.

Reply:
I've added some lines to test the declaration fragments, since that's what was crashing SymbolGraphGen.